diff --git a/baselines/baselines.py b/baselines/baselines.py
index bba806b..d8ccc83 100644
--- a/baselines/baselines.py
+++ b/baselines/baselines.py
@@ -1,5 +1,5 @@
 import gym
-from gym.wrappers import AtariPreprocessing
+#from gym.wrappers import AtariPreprocessing
 from stable_baselines3 import DQN
 from stable_baselines3.common.monitor import Monitor
 from stable_baselines3.common.vec_env import DummyVecEnv, VecVideoRecorder
@@ -36,9 +36,8 @@ def make_env():
     env = Monitor(env)
     # gym has a nice module for preprocessing Atari images to the specification of
     # the Mnih paper, however Pong-v0 has built in frame skip, so we need to handle it
-    # a different way
-    if config["env_name"] != "Pong-v0":
-        env = AtariPreprocessing(env, grayscale_obs=True)
+    # a different way, also the AtariPreprocessing module doesn't seem to output images
+    # like we need
     return env
 
 env = DummyVecEnv([make_env])
diff --git a/baselines/wandb/latest-run b/baselines/wandb/latest-run
index fcfb414..b3300ce 120000
--- a/baselines/wandb/latest-run
+++ b/baselines/wandb/latest-run
@@ -1 +1 @@
-run-20210903_050537-m6trvoch
\ No newline at end of file
+run-20210914_052331-1w5eksje
\ No newline at end of file
