diff --git a/baselines/baselines.py b/baselines/baselines.py
index 1af7734..bba806b 100644
--- a/baselines/baselines.py
+++ b/baselines/baselines.py
@@ -13,7 +13,7 @@ envs = []
 config = {
     "policy_type":"CnnPolicy",
     "total_timesteps":1000000,
-    "env_name":"Pong-v0",
+    "env_name":"PongNoFrameskip-v4",
 }
 
 # run configuration
diff --git a/baselines/wandb/latest-run b/baselines/wandb/latest-run
index fcfb414..d2858d9 120000
--- a/baselines/wandb/latest-run
+++ b/baselines/wandb/latest-run
@@ -1 +1 @@
-run-20210903_050537-m6trvoch
\ No newline at end of file
+run-20210911_152630-21x0ldyo
\ No newline at end of file
